pkg/client/dubbo/mapper.go

```diff
@@ -35,14 +36,25 @@ import (
 	"github.com/dubbogo/dubbo-go-proxy/pkg/config"
 )
 
+var mappers = map[string]client.ParamMapper{
+	constant.QueryStrings: queryStringsMapper{},
+	constant.Headers:      headerMapper{},
+	constant.RequestBody:  bodyMapper{},
+	constant.RequestURI:   uriMapper{},
+}
+
 type queryStringsMapper struct{}
 
+// nolint
 func (qm queryStringsMapper) Map(mp config.MappingParam, c client.Request, target interface{}) error {
```

Comment:
why isn't  the caller pointer?

Reply:
originally, I don't hope to modify anything inside the client.Request, but since I have to add back the body after mapping the request body, I think pointer might be better now.